src/Dialogs/Settings/Panels/TaskRulesConfigPanel.cpp

```diff
@@ -116,6 +119,24 @@ TaskRulesConfigPanel::Prepare(ContainerWindow &parent,
           altitude_reference_list,
           (unsigned)task_behaviour.ordered_defaults.finish_constraints.min_height_ref);
   SetExpertRow(FinishHeightRef);
+
+  AddSpacer();
+  SetExpertRow(spacer_3);
+
+  AddFloat(_("PEV start wait time"),
+             _("Wait time in minutes after Pilot Event and before start gate opens. "
+               "0 means start opens immediately."),
+             _T("%.0f min"), _T("%.0f min"), 0, 30, 1, false, UnitGroup::ALTITUDE,
```

Comment:
`UnitGroup::ALTITUDE` sounds wrong

Reply:
I agree. I forgot I did this hack. 

I hesitated to add another UnitGroup for `MINUTES` just for this use case though. Would you recommend introducing `UnitGroup::MINUTE` or rather use `UnitGroup::NONE` here? I don't remember exactly what was wrong now, but  `UnitGroup::NONE` didn't work here out of the box. I can collect more details on this if that's helpful.

Reply:
No new UnitGroup. XCSoar's units are for values where the user may choose different display units, but the code internally uses the base SI unit.
The first thing that's somewhat "wrong" with your code is that it doesn't use SI units; the SI unit for duration is seconds, not minutes. I'm not 100% sure if I want to demand you to switch to SI units - there may be good reasons to stay with minutes, but using SI units is never wrong, because it avoids a whole class of bugs.
The second thing that's wrong - if you're not going to use a UnitGroup, don't use the AddFloat overload that takes a UnitGroup parameter. Don't use UnitGroup::NONE.

Reply:
I have discovered `AddTime()`, which works with seconds. This was a good enough incentive to switch from minutes to seconds.

Reply:
I think you may need to adjust the help text. "in minutes" is redundant because the UI already displays the unit.